src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagController.cs

```diff
@@ -315,5 +363,69 @@ private static LogFormat ComputeLogFormat(IList<MediaTypeHeaderValue> acceptedHe
             }
             return LogFormat.None;
         }
+
+        private static ActionResult Result(
```

Comment:
@wiktork, do you see any value in applying the same capability to the metrics endpoint (where you can egress a snapshot of the metrics)? I feel that while it could be used in that manner, it's really more for scraping metrics rather than saving off an artifact.

Reply:
I don't think this will be worthwhile for metrics:
- Most current metrics systems will add dimensional data that we won't have here making the data much less meaningful.
- We should make it easy to correlate the metrics that get scraped with other artifacts we egress.